### PR TITLE
Backfill CUR

### PR DESCRIPTION
**Pull request by azure562**
- Add backfill historical CUR
- Update screenshots
- Add an FAQ entry

### CODE REVIEW

File: website/docs/amazon-web-services/backfill-cur.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+:::
+
+## CUR files
+
+The CUR files must follow the AWS standard:
+
+- The format of the billing files is CSV.
+
+- The billing files are compressed using `zip` or `gz`.
```
Comment (norosa) [inline website/docs/amazon-web-services/backfill-cur.mdx:56]: I think [7-Zip](https://www.7-zip.org/download.html) and [gzip](https://en.wikipedia.org/wiki/Gzip) are two different compression formats. It's probably better to specify `gzip`.

File: website/docs/amazon-web-services/invite-existing-aws-account.mdx [hunk 2]
```diff
@@ -11,133 +11,118 @@ You can add existing AWS accounts to DoiT International's consolidated billing u
 
 Required Permission: **Licenses Manager**
 
-You must be assigned access to the specific Billing Profile under which the domain is managed in order to invite an existing account.
+You need access to the Billing Profile which manages the domain.
```
Comment (norosa) [inline website/docs/amazon-web-services/invite-existing-aws-account.mdx:14]: I typically italicize terms like _Billing Profile_ when they carry special meaning as noun phrases (at least the first time I use them on a page). In this instance, perhaps, and in other instances like it, we could link the noun phrase to the appropriate page in the docs, e.g.:

- https://help.doit-intl.com/docs/invoices-and-payments/setting-up-a-new-billing-profile

That repurposes the link as a sort of definition of the term.

File: website/docs/amazon-web-services/invite-existing-aws-account.mdx [hunk 3]
```diff
@@ -11,133 +11,118 @@ You can add existing AWS accounts to DoiT International's consolidated billing u
...
 
-## Frequently asked questions
+After accepting the invitation in the AWS Console, you'll receive a final email about how to complete the IAM access for CloudHealth. See also [Set up CloudHealth](set-up-cloudhealth.mdx).
+
+## FAQ
 
 ### Why should I use the CMP for cost analytics when Amazon provides the AWS Cost Explorer?
 
-When you consolidate with the DoiT International billing organization, the [AWS Cost Explorer](https://aws.amazon.com/aws-cost-management/aws-cost-explorer/) can only report on [blended rates and costs](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/con-bill-blended-rates.html#Blended_CB). On the other hand, the CMP's reporting and analytics features work directly with your non-blended cost data.
+When you consolidate with the DoiT International billing organization, the [AWS Cost Explorer](https://aws.amazon.com/aws-cost-management/aws-cost-explorer/) can only report on [blended rates and costs](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/con-bill-blended-rates.html#Blended_CB), while the CMP's reporting and analytics features work directly with your non-blended cost data.
```
Comment (norosa) [inline website/docs/amazon-web-services/invite-existing-aws-account.mdx:90]: Is there another way to rephrase this that doesn't string so many subordinate clauses together into a single sentence? "On the other hand" might not be the best approach, but I do like the way it separates the paragraph into two easier-to-read sentences.

**Review by norosa — APPROVED** [review]

File: website/docs/amazon-web-services/invite-existing-aws-account.mdx (hunk 3 above)
Comment (azure562) [reply website/docs/amazon-web-services/invite-existing-aws-account.mdx:90]: it looks fine to me

File: website/docs/amazon-web-services/invite-existing-aws-account.mdx (hunk 2 above)
Comment (azure562) [reply website/docs/amazon-web-services/invite-existing-aws-account.mdx:14]: added link, not sure about italitics as it doesn't need to be emphasized that much

File: website/docs/amazon-web-services/backfill-cur.mdx (hunk 1 above)
Comment (azure562) [reply website/docs/amazon-web-services/backfill-cur.mdx:56]: from https://docs.aws.amazon.com/cur/latest/userguide/understanding-report-versions.html. "zip or gz = The type of compression applied to the report files."

**Pull request merged**